src/color-swatch/README.md

```diff
@@ -131,4 +137,35 @@ If you don’t, the `<html>` element will be used.
 	<input value="oklch(70% 0.25 138)" />
 </color-swatch>
 <button onclick='dynamic_editable.color = "oklch(60% 0.15 0)"'>Change color</button>
-```
\ No newline at end of file
+```
+
+## Reference
+
+### Attributes & Properties
+
+| Attribute | Property | Property type | Default value | Description |
+|-----------|----------|---------------|---------------|-------------|
+| `color` | `color` | `Color` &#124; `string` | - | The current color value. |
+| `value` | `value` | `string` | - | The current value of the swatch. |
+| `size` | - | `large` | - | The size of the swatch. Currently, it is used only to make a large swatch. |
+| `property` | `property` | `string` | - | CSS property to bind to. |
+| `scope` | `scope` | `string` | `:root` | CSS selector to use as the scope for the specified CSS property. |
+| `verbatim` | `verbatim` | `boolean` | `false` | ??? |
```

Comment:
Don't know how to describe this attribute correctly. 🤷‍♂️

Reply:
Just don't include it for now

Reply:
(and then remove the example)

Reply:
Done